agenta-backend/agenta_backend/services/evaluation_service.py

```diff
@@ -458,7 +458,7 @@ def _extend_with_evaluation(evaluation_type: EvaluationType):
 
 def _extend_with_correct_answer(evaluation_type: EvaluationType, row: dict):
     correct_answer = {}
```

Comment:
why don't we init to 
```suggestion
    correct_answer = {"correct_answer":""}
```